app/assets/stylesheets/calendar.css.scss

```diff
@@ -1,49 +1,15 @@
-// Place all the styles related to the shoes controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
-#shoes .calendar {
-  border-collapse: collapse;
-  width: 100%;
-
-  td, th {
-    font-family: "Lucida Grande", arial, helvetica, sans-serif;
-    font-size: 10px;
-    padding: 6px;
-    border: 1px solid #999;
+#calendar {
+  .calendar_content {
+    p {
+      font-size: 0.4rem;
+      text-align: center;
+      margin-bottom: 0;
+    }
+    td {
+      padding: 0 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0` instead of `0 0`